omise/classes/omise_charge_class.php

```diff
@@ -50,7 +50,7 @@ public function create($card_token)
     }
 
     /**
-     * @param string $offsite The parameter used to specify a bank to create Omise internet banking charge.
+     * @param string/array $offsite The parameter used to specify simple offsite type, or array of type and additional params
```

Comment:
Documentation comment need to change here.
`* @param string | array $offsiteType` or ` *@param mixed $offsiteType`

Reply:
? I've said it can be a string or array

Reply:
Right but string|array should be written in docs (not with /)

Reply:
I had no idea. We're not strict on this stuff anyway - you'll probably find loads of stuff like that around. Changed in my latest branch 👍